tests/app-template/package.json

```diff
@@ -41,6 +41,7 @@
     "@embroider/core": "workspace:*",
     "@embroider/router": "workspace:*",
     "@embroider/test-setup": "workspace:*",
+    "@embroider/virtual": "workspace:*",
```

Comment:
I think build needs it to, not just types. The errors i was running in to yesterday were because I didn't have a ./* Entry in the exports.

That's probably a separate issue tbh. Like.. why's a real package.json get involved with virtual files?

> How does ember-source solve this similar problem? 🤔

ember-source does a bunch of declare module, and requires a specific config. That could take the form of an import 'ember-source/types', or what we've moved to in the blueprints, where you specify ember-source/types in tsconfig#compilerOptions#types

We could do the same trick, but resolving the exports lookup thing would need to happen first.

Reply:
That said, idk if possible here, i'd like us to not have so many diffs between js and usages.

Some linting would also complain about importing from a package that is not declared.